src/errors.ts

```diff
@@ -10,7 +10,7 @@ export let errorCodes: IHash = {
     "only_factory_contract": "Only factory contract has access. The access can be to add new app or setup app level limit i.e, storage and bandwidth.",
     "no_app_space": "Your current app's storage or bandwidth limit has been consumed.",
     "no_user_space": "You have already consumed your storage or bandwidth limit.",
-    "non_trusted_forwarder_or_factory": "For meta-transaction, transaction should happen from valid factory or forwarder contract.",
+    "non_trusted_forwarder_or_factory": "For meta transaction, transaction should happen from valid factory or forwarder contract.",
```

Comment:
Shouldn't this be “a meta-transaction”? “Meta transaction” isn't grammatically correct since meta is part of a compound word.

Reply:
the reason I removed `-` is because [cleanMessage](https://github.com/arcana-network/storage/blob/dev/src/Utils.ts#L119) removes `-`

Reply:
It probably shouldn't.